docs/conf.py

```diff
@@ -41,7 +42,7 @@
 
 # General information about the project.
 project = u'Praekelt Ways of Working'
-copyright = u'2013, Praekelt Foundation devs and individual contributors'
+copyright = '%d, Praekelt Foundation developers and individual contributors' % date.today().year
```

Comment:
We should probably just remove "developers" here -- it's copyright the organization and individual contributors.

Not sure about automatically setting the copyright date, but maybe lets go with this unless anyone complains?